examples/example-node-api-client/client-credentials.ts

```diff
@@ -3,12 +3,18 @@ import { printTable } from 'console-table-printer'
 import inquirer from 'inquirer'
 import { ClientPool } from '@roll-network/api-client'
 import { SDKPool, InteractionType } from '@roll-network/auth-sdk'
-import config from './config.js'
+import baseConfig from './config.js'
+
+/*
+We have to use `roll-app` here, because client credentials flow requires an access token
+generated by browser oauth flow for the Roll admin user
+*/
+const config = { ...baseConfig, clientId: 'roll-app' }
```

Comment:
why not use env params here?

For our use case, we will use roll-app if we want to generate a token for the admin, since that is the owner of the clients and we need it to generate a new client secret.

For other clients, they may have an additional owner, or maybe we want to login as some other user for some reason. For example maybe I want to send a transaction from my personal Roll account via this example repo